src/installer.ts

```diff
@@ -30,6 +30,143 @@ interface INodeRelease extends tc.IToolRelease {
   lts?: string;
 }
 
+export enum Distributions {
+  DEFAULT,
+  CANARY,
+  NIGHTLY,
+  RC
+}
+
+export const distributionOf = (versionSpec: string): Distributions =>
+  versionSpec.includes('-v8-canary')
```

Comment:
Could you please change it to `switch or if block` for better readability.

Reply:
fixed